lib/katello/plugin.rb

```diff
@@ -265,6 +265,29 @@
       :onlyif => proc { |proxy| proxy.pulp_primary? }
   end
 
+  extend_page 'hosts/_list' do |context|
+    context.with_profile :content, _('Content'), default: false do
+      common_th_class = 'hidden-tablet hidden-xs'
+      common_td_class = "#{common_th_class} ellipsis"
+      use_pagelet :hosts_table_column_header, :name
+      use_pagelet :hosts_table_column_content, :name
+      add_pagelet :hosts_table_column_header, key: :subscription_status, label: _('Subscription status'), sortable: true, class: common_th_class, width: '10%'
+      add_pagelet :hosts_table_column_content, key: :subscription_status, class: common_td_class, callback: ->(host) { host_status_icon(host.subscription_global_status) }
+      add_pagelet :hosts_table_column_header, key: :status, label: _('Installable updates'), sortable: true, class: common_th_class, width: '15%'
+      add_pagelet :hosts_table_column_content, key: :status, class: common_td_class, callback: ->(host) { errata_counts(host) }
```

Comment:
I'm not 100% sure how this works, but isn't `status` a little bit vague?

Also I'm getting `the field 'status' in the order statement is not valid field for search` when trying to order by it